src/PROPOSAL/scattering/stochastic_deflection/bremsstrahlung/NaivBremsstrahlung.cxx

```diff
@@ -1,11 +1,22 @@
 #include "PROPOSAL/scattering/stochastic_deflection/bremsstrahlung/NaivBremsstrahlung.h"
 #include "PROPOSAL/Constants.h"
-#include <iostream>
+
 using namespace PROPOSAL;
 
 std::array<double, 2>
```

Comment:
`std::array` seems like a strange choice here. C++ normally encourages `std::pair` or `std::tuple` with `std::make_pair` and `std::make_tuple` for multiple return values.

Reply:
I don't see the point.
These is a homogeneous type containers which are faster than resolving the underlying data type. What advantage do I get by using a tuple?

Reply:
If this is an API where the two values have specific meaning and should all be the same, using a struct would probably be the best option. 

```
struct Deflection {
    double find_a_good_name_for_this1;
    double find_a_good_name_for_this1;
}
```

Reply:
Compiled code will basically be the same. So it's mostly a matter of test and consistency. I would favor the struct approach as you can give the fields a semantic meaning and you can document it using doxygen at the struct.

Reply:
Doesn't that imply that every stl container is unnecessary?
Maybe an enum should be created which defines the access to the elements explicitly. Otherwise I don't see the need to create a separate object for it.

Reply:
The struct approach will also give you better type safety, you cannot call a function that just happens to also return an array of two doubles in a context where the output of a deflection would be required.